.mailmap

```diff
@@ -17,6 +17,8 @@ Carl Fürstenberg <azatoth@gmail.com> <carl@excito.com>
 Charles McNulty <cmcnulty@kznf.com>
 Chris Rebert <github@rebertia.com>
 Chris Rebert <github@rebertia.com> <code@rebertia.com>
+Christian Oliff <christianoliff@pm.me>
+Christian Oliff <christianoliff@pm.me> <christianoliff@yahoo.com>
```

Comment:
@coliff Which email do you prefer? Each of your two commits have been done under a different one; these entries will normalize it.

I made this PR with the assumption you prefer `christianoliff@pm.me` but let me know if you prefer the other one & I'll reverse the PR.

Reply:
thanks @mgol  - christianoliff@pm.me would be perfect. thanks again!